### DISCUSSION TITLE
💻 Shorten our error messages

### DISCUSSION THREAD

**Discussion by MarleenGilsing** [hedyorg/hedy#5079] [Large Projects & Plans]
Our error messages might be longer than necessary.  For instance:
![image](https://github.com/hedyorg/hedy/assets/80678586/12dc6b6e-9002-4da7-a16a-3d8e3ce8fcd6)

This error message doesn't need 'at position 13'. I don't think students know what we mean by position 13 anyway and the mistake is highlighted nowadays, so the student will instantly see what they did wrong in line 1. 

I would even argue that 'The code you entered is not valid Hedy code.' is unnecessary information.

I think the shorter the message, the bigger change of a kid actually reading the message, so maybe we can take a look at the messages and tweak all the extra info.

**Comment by Felienne** [comment]
Great remark! I have made similar remarks in https://github.com/hedyorg/hedy/issues/4729 Let's discuss this at the hackathon?
